src/main/java/org/wildfly/security/http/HttpExchangeSpi.java

```diff
@@ -122,6 +122,14 @@ default SSLSession getSSLSession() {
     URI getRequestURI();
 
     /**
+     * Get the request relative path. This is the path relative to the context path. E.g.: for a request to
+     * <code>http://my.appserver.com/my-application/path/sub-path</code> this method is going to return <code>/path/sub-path</code>.
+     *
+     * @return the request relative path
+     */
+    String getRelativePath();
```

Comment:
OK, I have just used the same name we have in Undertow. But I agree, ```getRequestPath```  sounds better.